heroic-core/src/main/java/com/spotify/heroic/cluster/CoreClusterManager.java

```diff
@@ -100,8 +103,9 @@ public CoreClusterManager(
       LocalClusterNode local,
       HeroicContext context,
       @Named("topology") Set<Map<String, String>> expectedTopology,
-
-      final QueryReporter reporter) {
+      final QueryReporter reporter,
+      @Named("cluster") UsageTracking usageTracking
```

Comment:
why is this cluster? shouldn't it be `usageTracking`?

Reply:
It's the cluster version of the `UsageTracking` injected object, which distinguishes it from other Providers for `UsageTracking`.

Reply:
Actually, I realized that's not needed at all, I can just remove the Provides from the cluster module. Will update in the scheduled PR.